roles/openstack-setup/tasks/users.yml

```diff
@@ -15,6 +15,8 @@
                     login_password={{ secrets.admin_password }}
                     insecure={{ insecure|default(omit) }}
   with_items: "{{ keystone.services }}"
-  when: "{{item.name}}.enabled is undefined or {{item.name}}.enabled|bool"
+  when: 
+    - (item.name.enabled is undefined or item.name.enabled|bool)
```

Comment:
question here: so this task will run if enabled == true, or if enabled is not defined? What's the reason we want to run this if enabled is not defined?